tests/lora/test_long_context.py

```diff
@@ -115,6 +115,7 @@ def lora_llm(long_context_infos):
                    long_lora_scaling_factors=tuple(scaling_factors),
                    max_num_batched_tokens=4096 * 8,
                    tensor_parallel_size=4,
+                   disable_async_output_proc=True,
```

Comment:
Can you add a `FIXME` here as a reminder to fix the test when async output processor is enabled?

Reply:
Done, and please add `/ready` ,thanks~